src/components/time-on-target-chart/TimeOnTargetChart.svelte

```diff
@@ -31,63 +31,65 @@
+            <input type="text" bind:value={shipName} maxlength="26" class="text-input large-text-input p-0 has-text-centered" placeholder="Ship Name Here">
+    
+            <h3 class="mt-2 mb-0">Ammo Selection</h3>
+            <select class="dropdown-select mt-1" bind:value={ammo}>
+                <option value="Basic">Basic</option>
+                <option value="Standard">Standard</option>
+                <option value="Premium">Premium</option>
+            </select>
+    
+            <h3 class="mt-2 mb-0">Accuracy</h3>
+            <select class="dropdown-select mt-1" bind:value={accuracy}>
+                <option value="100">100%</option>
+                <option value="75">75%</option>
+                <option value="50">50%</option>
+            </select>
+    
+            <button class="download-button" on:click={chartToImage}>Download Chart <br> 
+                                                                        as PNG</button>
+    
+        </div>
     </div>
-    <div>
-        <div class="create-a-chart-container">
-            <div class="ml-2 mt-3">
-                <div class="has-font-20 mt-1">{shipName || "AX Ship"}</div>
-                <div class="has-font-20 mt-1">
-                    {`${$selectedDistributor.size}${$selectedDistributor.rating}
-                     distributor (${distributorEngineering()})`
-                    }
-                </div>
-                <div class="has-font-20 mt-1">{ammo} ammo</div>
-                <div class="has-font-20 mt-1">{accuracy}% accuracy</div>
-                {#if haveGaussSelected}
-                    {#if toggleMm}
-                        <div class="has-font-20 mt-1">{delay}ms ({delayInMm} Millimechs)</div> 
-                    {:else}
-                        <div class="has-font-20 mt-1">{delay}ms ({bpm} BPM)</div>
-                    {/if}
-                {:else}
-                    <div></div>
-                {/if}
+    
+    <div id="customChart" class="create-a-chart-container">
+        <div class="ml-2 mt-3">
+            <div class="has-font-20 mt-1">{shipName || "AX Ship"}</div>
+            <div class="has-font-20 mt-1">
+                {`${$selectedDistributor.size}${$selectedDistributor.rating}
+                    distributor (${distributorEngineering()})`
+                }
             </div>
+            <div class="has-font-20 mt-1">{ammo} ammo</div>
+            <div class="has-font-20 mt-1">{accuracy}% accuracy</div>
+            {#if haveGaussSelected}
+                <div class="has-font-20 mt-1">{delay}ms ({bpm} BPM)</div>
+            {:else}
+                <div></div>
+            {/if}
+        </div>
 
-            <div class="graphic mt-4 is-justify-content-center">
-                {#each THARGOID_TYPES as interceptor}
-                    <InterceptorData bind:accuracy bind:ammo interceptor={interceptor.name}/>
-                {/each}
-            </div>
+        <div class="graphic mt-4 is-justify-content-center">

```

Comment:
Looks like the line endings were changing types here potentially? We should make sure that there is a consistent setting here so there aren't constant unnecessary changes.